css/sidebar.css

```diff
@@ -0,0 +1,39 @@
+.sidebar {
+  position: fixed;
+  width: 200px;
+  height: 500px;
```

Comment:
Can give sidebar height relative to viewport height so that it will look good on both larger and smaller screen devices.